tests/test_api.py

```diff
@@ -42,3 +44,31 @@ def test_api_demo():
                 print(f"saw job {m.group('jobid')} command {m.group('command')}")
     else:
         print("submission failed: output:\n", out1)
+
+
+from jobsub_api import submit, q
+
+
+def test_fancy_api_demo():
+    group = "fermilab"
+    testdir = os.path.dirname(__file__)
+    try:
+        job1 = submit(
+            group=group, executable=f"{testdir}/job_scripts/lookaround.sh", verbose=1
+        )
+        print(f"submitted job: {job1.id}")
+        print(f"submit output:\n=-=-=-=-=\n {job1.submit_out}")
+        print(f"\n=-=-=-=-=\n")
+        qjobs = q(job1.id, group=group)
+        for qjob in qjobs:
+            print(f"saw job {qjob.id} status {str(qjob.status)} ")
+        rs = job1.fetchlog(destdir="/tmp/test_fetch", verbose=1)
+        print(f"fetchlog says: {rs}")
+        sb = os.stat("/tmp/test_fetch/lookaround.sh")
+        assert time.time() - sb.st_mtime < 5
+        os.system("rm -rf /tmp/test_fetch")
+        data = job1.q_long()
+        print(f"after update, status: {str(job1.status)}")
+        assert "ClusterId" in data
+    except RuntimeError:
```

Comment:
I'm not sure this is necessary: If there's a `RuntimeError` while running the test, it'll just fail, which is what we want.

Reply:
I think I had some debug printing in there at one point...

Reply:
I think the real reason was, I wanted this to be a steal-able example of using the API, and the things that
in a shell script exit non-zero raise RunTimeError in the API.   So I think I'll re-add the print, and leave it in
for just that reason.